### PR TITLE
Add org.gnome.Sudoku

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Upstream discussion: https://gitlab.gnome.org/GNOME/gnome-sudoku/issues/32

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
bot, build org.gnome.Sudoku

File: org.gnome.Sudoku.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* dconf */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [{
+            "name": "libgee",
```
Comment (mcatanzaro) [inline org.gnome.Sudoku.json]: This should be in the 3.32 runtime. Try removing it!

Comment (bilelmoussaoui) [reply org.gnome.Sudoku.json]: Wasn't aware of that, will remove it.

Comment (bilelmoussaoui) [reply org.gnome.Sudoku.json]: Still the build fails due to missing release tags.

**Comment by bilelmoussaoui** [comment]
I have sent a MR upstream here to fix the release tags https://gitlab.gnome.org/GNOME/gnome-sudoku/merge_requests/9

**Comment by nedrichards** [comment]
@bilelmoussaoui per https://gitlab.gnome.org/GNOME/gnome-sudoku/merge_requests/9#note_522037 I'd be happy to take the patch here and move on with our lives whilst we work out a good way of doing release tags upstream with the project as a whole.

**Comment by bilelmoussaoui** [comment]
@nedrichards I can do send the changes upstream, I will add the patcher soon

**Comment by bilelmoussaoui** [comment]
bot, build org.gnome.Sudoku

**Comment by nedrichards** [comment]
> • attribute-missing     : <release> has no timestamp

validation doesn't like your date format.

**Comment by bilelmoussaoui** [comment]
> > • attribute-missing     :  has no timestamp
> 
> validation doesn't like your date format.

It's asking for a timestamp attribute too https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html. The date format is correct per the specs

**Comment by nedrichards** [comment]
Indeed! That, er, shouldn't be a blocker.

**Comment by bilelmoussaoui** [comment]
@barthalion any idea? :)

File: appdata-release-tag.patch [hunk 2]
```diff
@@ -0,0 +1,12 @@
+--- a/data/org.gnome.Sudoku.appdata.xml.in	2019-06-04 14:48:52.126188662 +0200
++++ b/data/org.gnome.Sudoku.appdata.xml.in	2019-06-04 14:47:34.211892440 +0200
+@@ -46,6 +46,9 @@
+     <kudo>ModernToolkit</kudo>
+     <kudo>UserDocs</kudo>
+   </kudos>
++  <releases>
++    <release version="3.32.0" date="2O19-03-11" />
```
Comment (barthalion) [inline appdata-release-tag.patch]: This might be hard to spot, but it actually says `2O19` with capital O, not zero.

**Comment by barthalion** [comment]
`2019 vs 2O19`, see my comment under the patch.

**Comment by nedrichards** [comment]
bot, build org.gnome.Sudoku

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gnome.Sudoku

**Pull request closed without merging**